File: myonlinefm/src/main/java/com/sec/myonlinefm/classificationprogram/data/DemandChannelPattern.java
package com.sec.myonlinefm.classificationprogram.data;

import java.util.List;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/17.
 */

public class DemandChannelPattern {

    private List<DemandChannel> demandChannelsList;
    private DemandChannel currentDemandChannel;

    public void setDemandChannelsList(List<DemandChannel> demandChannelsList) {
        this.demandChannelsList = demandChannelsList;
    }
    public List<DemandChannel> getDemandChannelsList() {
        return demandChannelsList;
    }

    public void setCurrentDemandChannel(DemandChannel currentDemandChannel) {
        this.currentDemandChannel = currentDemandChannel;
    }

    public DemandChannel getCurrentDemandChannel() {
        return currentDemandChannel;
    }
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/classificationprogram/data/DemandChannelPattern.kt
package com.sec.myonlinefm.classificationprogram.dataimport

import com.sec.myonlinefm.classificationprogram.data.DemandChannel

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/17.
 */
class DemandChannelPattern {
    internal var demandChannelsList: MutableList<DemandChannel?>? = null
    internal var currentDemandChannel: DemandChannel? = null
    fun setDemandChannelsList(demandChannelsList: MutableList<DemandChannel?>?) {
        this.demandChannelsList = demandChannelsList
    }

    fun getDemandChannelsList(): MutableList<DemandChannel?>? {
        return demandChannelsList
    }

    fun setCurrentDemandChannel(currentDemandChannel: DemandChannel?) {
        this.currentDemandChannel = currentDemandChannel
    }

    fun getCurrentDemandChannel(): DemandChannel? {
        return currentDemandChannel
    }
}